api.h

```diff
@@ -0,0 +1,64 @@
+#ifndef API_H
+#define API_H
+
+/*************************************************
+ * Name:        XMSS_crypto_sign_keypair
+ *
+ * Description: Generates public and private key.
+ *
+ * Arguments:   - uint8_t *pk: pointer to output public key (allocated
+ *                             array of CRYPTO_PUBLIC_KEY bytes)
+ *              - uint8_t *sk: pointer to output private key (allocated
+ *                             array of CRYPTO_SECRET_KEY bytes)
+ *
+ * Returns 0 (success), -1 otherwise
+ **************************************************/
+int crypto_sign_keypair(unsigned char *pk, unsigned char *sk);
+
+/*************************************************
+ * Name:        XMSS_crypto_sign
+ *
+ * Description: Computes signature.
+ *
+ * Arguments:   - uint8_t *sm:   pointer to output signature (of length CRYPTO_BYTES)
```

Comment:
Please have the order of the arguments in the documentation match the order of the arguments in the function signature.